sympy/functions/combinatorial/factorials.py

```diff
@@ -357,6 +359,10 @@ class factorial2(CombinatorialFunction):
     @classmethod
     def eval(cls, arg):
         # TODO: extend this to complex numbers?
+        check = arg.is_real
+
+        if check == False:
+            return gamma(arg)
```

Comment:
This isn't the correct double factorial. A formula for double factorial in the complex plane can be found on [Wolfram site](http://functions.wolfram.com/GammaBetaErf/Factorial2/02/).